parlai/scripts/self_chat.py

```diff
@@ -48,7 +48,10 @@ def setup_args(parser=None):
     )
     parser.add_argument('--outfile', type=str, default='/tmp/selfchat.json')
     parser.add_argument(
-        '--format', type=str, default='json', choices={'parlai', 'json'}
+        '--format', type=str, default='jsonl', choices={'parlai', 'jsonl'}
+    )
+    parser.add_argument(
```

Comment:
Is there particularly a reason to need to vary the indent?

Reply:
yes, the indent makes the file look pretty but absolutely kills processing via `json.loads`